## Notes for next meeting
### Consensus Sequence Blast Results 
#### 700 bp consenus sequence 
##### Using MSA 
- 285 bp long
- top matches are symbiodinium w/ 95% identity 
#### 1500 bp consensus sequence 
##### Using MSA
- 840 bp long 
- top matches are cladocopium w/ 100% identity 
#### Blasting both 700 and 1500 together (2 sequence Blast)
##### Using MSA 
- top 4 matches were symbiodinium w/ 95% identity 
- rest were cladocopium w/ 90% identity 
- one Brevolium w/ 92% identity which was exciting to see 
#### I created a merge/consensus between the 700 and 1500 consensus sequence 
##### Using MSA
- top matches are all cladocopium w/ 100% identity for at least the top 10 rows 
### How past literature determined symbiont identity
#### LaJeunesse & Thornhill 2011 
- amplified psbA in samples they already knew ITS2 ID, identified the psbA symbionts by checking which ITS2 type each sample already had.
	- need to look to see how they got ITS2 ids when symportal wasn't a thing yet 
#### Marzonie 2024 
- sanger sequences were manually trimmed and aligned using DECIPHER (Wright, 2016).
	- read this [10.32614/RJ-2016-025](https://doi.org/10.32614/RJ-2016-025) 
	- wonder if decipher will produce a different result than my msa results 
- paired ITS2 IDs from SymPortal with psbA sequences from the same samples to confirm the symbiont identity
- used ClusterW for alignment
#### Hume et al. 2015 
- used ITS2 identities from DGGE/cloning and then sequenced psbA ncr from the same samples to confirm
- used ClusterW for alignment 
##### next steps 
- need to look at papers that solely use psbA for symbiont ids. 

## 11/14/2025 w/ Sarah 

### Next Steps 
- reverse complements Reverse strand to forward strand and create a consensus sequence to BLAST- see if query results change/have more agreement 
- align 700 bp and 1500 bp bands together to see if they are the same sequences
- determine in 1500 bp and 700 bp are mapping to different regions of the top query since the number one hit was the same 
- determine how LaJeunesse et al. 2011 identified symbiont identity from BLAST or other means 
	- Caroline suggests checking supplemental and rereading paper to determine if I can find any questions/answers now that I have better understanding of my project material 
### sequencing results 
- none of my samples came back good quality 
#### nonspecfic: overlapping peaks of roughly equal intensity
AT0390151_T3_36_700bp_R
- peaks get weird around 600 bp 
AT0390152_T3_36_700bp_F
- begins with a lot fo background noise but clears up around 300 bp 
AT0390153_T3_38_1500bp_R
- ends with a lot of background noise and wonky peaks 
#### poor quality: low averager intensity, high background noise, signal gradually dimishes 
AT0390154_T3_38_1500bp_F
- after ~700 bp the signal becomes crazy but this might hold true for nonspecific binding 
AT0390155_T3_36_300bp_R
- there is a crap ton of background noise and also peaks dimish before 330 bp but the files keeps going so maybe that has something to do with the poor score also 
#### no primer: 
AT0390156_T3_36_300bp_F
- this is the first one I did so I doubt I just forgot I must of had no liquid in my tip when I put it in; will do better to stay on top of this next time 
- ton of sequencing noise not sure if I can use this one 

### BLAST Results 
700 bp came back as psbA Yippee
- even more awesome, the forward read had a 100% match which is exciting! 
1500 bp also returned psbA 
- however a large portion of this sequence was clouded by background noise so only blasted a small portion
300 bp came back as a coral bcl-2 gene? which is crazy (family of genes that regulate programmed cell death)
## 11/7/2025 with Sarah & Caroline 
Sigma Xi scholarship due March 15th - create reminder for Jan 

#### Sequencing Primers  
https://www.genewiz.com/public/resources/sample-submission-guidelines/sanger-sequencing-sample-submission-guidelines/technical-notes#sanger-sequence

- sequencing primer is just pcr primer 
- can send sequences back that might not be in the oritentation we expect 
- send out two replicates- one with forward and one with reverse primer 
#### add to ordering form 
- figure out if I need more polymerase 
- agar 
- tbe buffer 

## 10/23/2025 w/ Sarah and Caroline 
### Sample prep for sequencing 
Sample: 1 (cleaned sample of tube 1_6E from 9.2.2025) (300 bp)
Concentration: 8.62 ng/μL

Sample: 2 (cleaned sample of tube 2_4B from 9.2.2025) (1500 bp)
Concentration: 12.0 ng/μL

Sample: 4 (cleaned sample of tube 1_6B from 9.2.2025) (700 bp)
Concentration:  0.600 ng/μL
#### Next steps 
- try cleaning the other pcr sample where a 700 bp band is present
- chill ethanol before washing 
- do 15 uL of elution instead of 40 uL

### WDHOF application 
rough draft done woot woot
### Next Steps 
- submit WDHOF by Thursday 
- email Leah on Monday if she hasn't add the genewiz box by then 
- figure out C1V1 = C2V2 ratios for cleaned samples 


## 10/20/2025 w/ Caroline 
###  Testing out Vacuum Dryer
https://www.genewiz.com/public/resources/sample-submission-guidelines/sanger-sequencing-sample-submission-guidelines/sample-preparation#sanger-sequence 
- ez sanger wants 2ng/uL for conc and they want 10 uL of sample 
- need to vacuum filtrate samples down to about 12 uL and then qubit them 
	- need to qubit samples before vacuum filtering 

## 10/17/2025 w/ Sarah & Caroline
### Gel Questions 
- I am not sure what to do next regarding my gel 
- the 300bp and 1500 bp bands cleaned but the 700 bp band did not
	- Sarah thinks that I used too much elution in my final step of the bead cleanup so I diluted my samples by 2. update protocol so that the final elution step requires less than or equal to the amount of starting unclean PCR sample used.
### To do:
- qubit cleaned 700 bp
	- if concentration is low then work with Caroline to reconcentrate it 
- Look at the sanger sequences tubes ordered to determine information needed to send them out for Sequencing 
- Goal is to send out sequences next week using the drop box next to Fish lab on the 6th floor
- Email Leah about added drop box to her Genewiz account 
- finish rough draft of WDHOF proposal byh friday- share google doc with Caroline, Brooke, and Sarah to review 

### Submitting samples (Genewiz)
https://www.genewiz.com/public/resources/sample-submission-guidelines/sanger-sequencing-sample-submission-guidelines
I am a little confuse- they keep talking about universal genewiz primers but I am not sure why? primers are for amplification but I already did that? why do they need primers? 
- sequencing primers not pcr primers- we will use the genewiz universal primer 

## 10/10/2025 w/ Sarah & Caroline
### WDHOF Proposal Plan 
- should write it about going on the belize trip in May. 
- put $500 for your flight and  $400 do get dive gear for the caribbean (wetsuit/booties) 
- the rest of the money can be for research supplies like tubes and stuff
- should say that I am are currently looking at symbiont communities in Panama and want to contrast them with Belize
	- Belize experienced worse bleaching than panama- so you'll look at the difference between the two countires 
### 10.8.2025 Gel Issue 
- Sarah and Caroline believe I left the gel the microwave for tool long, so that most of the liquid evaporated making my gel more than 1.5% agarose
- Will remake a gel with Sarah and Caroline present to make sure I am pulling the gel out at the right time from the microwave 
### Goldwater Scholarship 
https://goldwaterscholarship.gov/
- Sarah thinks I should apply for this scholarship 
- due date January 30, 2026
- Dr. Rae Mansfield is out representative, need to email her about internal deadlines 
### To do 
- Look at genewiz website to determine how to send out samples for sequencing 
- begin wdhof project proposal 
- finish add last touches to poster for bio retreat 

## 10/3/2025 w/ Sarah & Caroline
### Sanger sequencing company 
Genewiz- Azenta Life Sciences 
https://www.genewiz.com/public/services/sanger-sequencing/sanger-ez
### To Do
- meet with caroline on monday to run through bead clean up protocol
- when get sequences back can use local blast to figure out if they are psbA sequences

## 9/25/2025 w/ Sarah & Caroline
### Sarahs throughts for Sigma Xi application 
- Will wait for membership approval and try applying for the grant in spring 
- sigma xi should pay for tarvel for collecting samples 
- frame proposal about continuing research to make to make it more exciting
### Next steps: Sequencing subset of samples 
- sequence 300 bp and 700 bp band seperately [9.2.2025 - Troubleshooting](../../PAN_BDT/psbA/9.2.2025%20-%20Troubleshooting.md)
	- 300 bp band from 6 uL sample 1 at 44.7 
	- 700 bp band from 6 uL sample 1 at 53.6 
		- need to do a size selection with beads to remove 300 bp band 
	- 1500 bp band from 4 uL sample 2 at 53.6
For bead clean up do 1.8x for 300 bp and .6x  for 700 bp sample (https://www.beckman.com/reagents/genomic/cleanup-and-size-selection/pcr/bead-ratio) 
### main concerns with psbA right now 
- why my samples arent all showing a 700 bp band
- why a majority of my samples actually show the 300 bp band and not the 700 bp band
	- is it because my host biomass is outweighing the symbiont biomass- so my host DNA is overwhleming the symbiont DNA? 
- Caleb says there is the possibility of there being two psbA seuqences in my samples- does that mean two different symbionts or one symbiont with mutliple psbA regions (is that possible)

## 9/17/2025 w/ Caroline & Brooke 
### To Do 
- draft email to Caleb (CalebCButler@txstate.edu)- post doc who's phd advisor was Todd LaJeunesse
	In my email I need to include:
		- briefly introduce myself include my lab  
		- upload most recent image (make sure to clearly annotate it)
		- explain what I am seeing in my gel and what I am concerned about
		- explain the brevolium genome database searching I did and why I think the 700 bp is my psbA region and ask for his opinion
#### Caroline's tips: 
- keep major question in bold in its own sentence right after introducing myself- they do not want to be search the emailing for it. 
- Keep it straight foward do not over explain yourself, they are also a coral biologist and if they need more detail it is better they ask then bore them with a long email 
- always can ask follow up questions later- do not ask too much from them at once 

### Caleb's Email Response notes 
Recieved on 9/18
- psbAncr can show intragenomic variation, usually less pronounced than ITS2
	- Intragenomic variation: multiple slightly different copies of the psbA ncr exist within a single symbiont genome; differences are usually small (SNPs or indels) and typically do not produce separate gel bands.
- Smaller bands (e.g., 300 bp) may result from PCR artifacts or host DNA amplification; in rare cases, they could reflect multiple psbA variants
- The expected psbAncr amplicon size is generally 700–1400 bp; the 700 bp band is likely the correct target so he suggests to sequence it and see what that yields 
## 9/10/2025 w/ Caroline & Brooke
### Questions I want to ask 
- any recommendations of how I should balance lab work with grant writing? I am struggle to stay on top of everything especially with my coursework
	- Caroline suggests to prioritize writing since those have hard deadlines opposed to personal (soft) deadline
### Brevolium Reference Genome 
#### Local Blast Attempt
https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_965643015.1/ (GCA_965643015.1)
- use local blast to search primer sequence against brevolium genome, however the primers came up with too many hit for this to yield any effective results. 
#### Searching published Brevolium psbAncr sequences
- Nikea did some digging on the NCBI database and found a list of published psbA ncr regions from Brevolium (https://www.ncbi.nlm.nih.gov/nuccore/?term=psbA+minicircle%3B+chloroplast+Breviolum)
	- one of them match the paper that originally used the Thornhill & Lajuenesse 2011 primers on a DLAB sample 
		- https://www.ncbi.nlm.nih.gov/nuccore/MH756051.1 Acession number: MH756051
	- 11 DCYL seq that range from 720 bp - 969 bp 
	- 2 MMEA seq that are ~1130 bp
	- 1 DLAB seq that is 1,454 bp
	I feel like the above numbers and enough to confirm that the 700 bp band in my gels is most likely my samples psbA region since the psbAncr sequences are spanning such a wide array of bp counts
## 9/8/2025 w/ Sarah and Nikea
### Interpreting Gel Results 
#### 300 bp & 700 bp Bands
My most recent gel ([9.2.2025 - Troubleshooting](../../PAN_BDT/psbA/9.2.2025%20-%20Troubleshooting.md)) show a ~300 bp band and a ~700 bp band. However, the paper I got my primer from shares that the psbAncr region was between 500-600 bp. 
- Sarah believe the next step is to figure out whay the 300 bp band is and if the 700 bp bands is actually the psbA region?
	- she says that we can perhaps reach out to Caleb in the Fuess lab as he has done psbAncr before see if he has thoughts on what the 300 bp band is and whether the ~700 is the target amplicon
	- or we can just sequence the 700 bp band and see, would just have to do a bead cleanup to get the 300 bp out my PCR samples
- Nikea says to use a genome browser to figure out what the lengths usually are in Breviolum since psbA is super conserved usually. 
	- she also said something about trying a local blast? need to ask more details on that since I am still new to explore genomic databases 
#### Non-specific Binding 
As I have been increasing my DNA load my non-specific binding has dissapeared. 
	**Sarahs thoughts on why:** raising the concentration is reducing the non-specific binding because you have a lot of non-target DNA (ie coral dna, microbe dna) in the samples and the primers aren't finding the right DNA. Therefore by increasing DNA load the primers can finally find the correct binding sites as there is more symbiont DNA. 