*trying to purify another 700 bp sample since the 700 bp sample from 10.6.2025 concentration was low*
sample: **1_6C:** T3_36 w/ 6 uL at temp C (want 700 bp band)

## 11/6/2025 Gel & Qubit Results 

### quibit 
Sample: **1_6C:** T3_36 w/ 6 uL at temp C (want 700 bp band)
Concentration: 4.48 ng/μL
### sending out sequence math 
*need to make 10 ul of 2 ng/ul*
Sample: **1_6C:** T3_36 w/ 6 uL at temp C (want 700 bp band)
	Concentration: 4.48 ng/μL
	V1 = C2(V2)/ C1 = (2 x 10)/4.48
		need:
		4.5 uL DNA
		5.5 uL water 

## 11/10/2025 sending out samples for sequencing 
need to create primer stock for sequencing that is 5 uL at 25 pmol. 
1 pmol/uL = 1 uM
	ex. 1 uM = 1 umol/L = 1 nmol/ml = 1 pmol/ul

so if I want 5 uL at 25 pmol then I want 5 pmol/ul = 5 uM

C1V1 = C2V2 
V1 = (C2V2)/C1 
V1 = (5uM x 5 uL)/10uM
V1 = 2.5 uL 

Need 2.5 uL of 10uM primer stock per sequence
*and need to add 2.5 uL of water to sample to make it total to 5 uL* 
### Bar code: AT03901558
Sample: 1 (cleaned 300 bp sample from 10.6.2025- originally tube 1_6E from 9.2.2025) 
	Concentration: 8.62 ng/μL
	V1 = C2(V2)/ C1 = (20)/8.62
		need:
		2.3 uL DNA
		7.7 uL water + 2.5 from primer dilution = 10.2 uL water 
		2.5 uL primer from 10 uM stock
		USED FORWARD PRIMER 

### Bar code: AT03901558
Sample: 1 (cleaned 300 bp sample from 10.6.2025- originally tube 1_6E from 9.2.2025) 
	Concentration: 8.62 ng/μL
	V1 = C2(V2)/ C1 = (20)/8.62
		need:
		2.3 uL DNA
		7.7 uL water + 2.5 from primer dilution = 10.2 uL water 
		2.5 uL primer from 10 uM stock
		USED REVERSE PRIMER 

### Bar code: AT03901557
Sample: 2 (cleaned 1500 bp sample from 10.6.2025- originally tube 2_4B from 9.2.2025)
	Concentration: 12.0 ng/μL
	V1 = C2(V2)/ C1 = (20)/12.00
		need: 
		1.7 uL DNA 
		8.3 uL water + 2.5 from primer dilution = 10.2 uL water 10.8 uL 
		2.5 uL primer from 10 uM stock
		USED FORWARD PRIMER

### Bar code: AT03901557
Sample: 2 (cleaned 1500 bp sample from 10.6.2025- originally tube 2_4B from 9.2.2025)
	Concentration: 12.0 ng/μL
	V1 = C2(V2)/ C1 = (20)/12.00
		need: 
		1.7 uL DNA 
		8.3 uL water + 2.5 from primer dilution = 10.2 uL water 10.8 uL 
		2.5 uL primer from 10 uM stock
		USED REVERSE PRIMER

### Bar Code: 

Sample: 3 (cleaned 700 bp sample from 11.5.2025- originally tube 1_6C from 9.2.2025) 
	Concentration: 4.48 ng/μL
	V1 = C2(V2)/ C1 = (2 x 10)/4.48
		need:
		4.5 uL DNA
		5.5 uL water + 2.5 uL from primer dilution = 8 uL 
		2.5 uL primer from 10 uM stock
		 USED FORWARD PRIMER

### Bar Code: 
Sample: 3 (cleaned 700 bp sample from 11.5.2025- originally tube 1_6C from 9.2.2025) 
	Concentration: 4.48 ng/μL
	V1 = C2(V2)/ C1 = (2 x 10)/4.48
		need:
		4.5 uL DNA
		5.5 uL water + 2.5 uL from primer dilution = 8 uL 
		2.5 uL primer from 10 uM stock
		 USED REVERSE PRIMER
# All done in the Post PCR Room
## III. Purification with ampure beads
https://www.beckman.com/reagents/genomic/cleanup-and-size-selection/pcr/bead-ratio
### Purification Preparation
- obtain, label, and cross-link new strip tubes start with the manufacturer protocol using 1.8X-1.0X DNA to bead ratio and 10uL-25uL PCR product 
- ratio of beads will change the size you select for 
	*may need to re-clean samples if gel images show that multiple bands were not removed*
	- *1.0x will get rid of <200 bp dimers, 1.8X will get rid of dimer <100 bp* 
- all calculations can be done here: [https://docs.google.com/spreadsheets/d/1O_NJCFvnBztKm_G88Sx-gEKD7CwR44iEaRjyxS_N32E/edit?gid=1947158502#gid=1947158502](https://docs.google.com/spreadsheets/d/1O_NJCFvnBztKm_G88Sx-gEKD7CwR44iEaRjyxS_N32E/edit?gid=1947158502#gid=1947158502) 

### Purification 
1. make fresh 80% ethanol in a 50mL tube (label and parafilm when not in use) - put in fridge to chill
    - paste filled out table here

| Number of Samples | 80% EtOH for each sample (uL) | Total 80% EtoH needed (uL) | Volume 100% EtOH (uL) | Volume H2O (uL) |
| ----------------- | ----------------------------- | -------------------------- | --------------------- | --------------- |
| 1                 | 360                           | 360                        | 288                   | 72              |
|                   |                               | in mL                      |                       |                 |
|                   |                               | 0.36                       | 0.288                 | 0.072           |

2. Determine whether or not a plate transfer is necessary. If the PCR reaction volume multiplied by 2.8 exceeds the volume of the PCR plate, a transfer to larger tubes is required. 
3. Gently shake the Clean NGS Mag PCR Clean-up aliquot to resuspend any Magnetic particles that may have settled.
    1. Add CleanNGS Mag PCR Clean-up volume table below:

| Bead Concentration | PCR volume (uL) | Added beads volume (uL) | Total # Samples | Total Bead Volume (uL) |
| ------------------ | --------------- | ----------------------- | --------------- | ---------------------- |
| 0.6                | 20              | 12                      | 1               | 12                     |

**Note:** The volume of CleanNGS Mag PCR Clean-up for a given reaction can be determined from the following equation:  
_(Volume of Mag Beads per reaction) = (Bead Concentration) x (PCR Reaction Volume)_

3. Mix reagent and PCR reaction thoroughly by pipette mixing 5 times.
4. Incubate the mixed samples for 5 minutes at room temperature for maximum recovery. 
	*This step allows the binding of PCR products 125bp (based on concentration) and greater to the Magnetic beads. After mixing, the color of the mixture should appear homogenous.*
5. Place the reaction plate onto a 96 well Magnet Plate for 3 minutes or wait until the solution is clear.
	*Wait until the solution is clear before proceeding to the next washing step. Otherwise there may be beads loss.*
6. Aspirate the cleared solution from the reaction plate and discard
	*This step must be performed while the reaction plate is placed on the 96 magnetic plate. Avoid disturbing the settled magnetic beads. If beads are drawn into tips, leave behind a few microliters of solution.*
7. Dispense **180 uL of 80% ethanol** to each well of the reaction plate and incubate for **1 min** at room temperature. 
8. Aspirate out the ethanol and discard. Repeat for a total of two washes. 
	*It is important to perform these steps with the reaction plate on a 96 well Magnetic Plate. Do not disturb the settled Magnetic beads.*
    1. Remove all of the ethanol from the bottom of the well to avoid ethanol carryover. 
	    *Bump pipette tip up to 200 uL, may need to use p20 multichannel*
    2. **NOTE:** *A 5 min air dry at room temperature is recommended for the evaporation of the remaining traces of ethanol.* **Do not overdry the beads** *(the layer of settled beads appears cracked) as this will significantly decrease elution efficiency.*
9. Take off the plate from the Magnetic plate, add 15 uL of elution buffer (Reagent grade water, TRIS-HCl pH 8.0, or TE buffer) to each well of the reaction plate and pipette mix 5 times.
    *mix until homogeneous and there are no beads on tube wall*
10. Incubate at room temperature for 10 minutes.
11. Place the plate on a magnetic separation device to magnetize the CleanNGS particles. 
12. Incubate at room temperature until the CleanNGS particles are completely cleared from solution.
13. Transfer the cleared supernatant containing purified DNA and/or RNA to a new (RNase-free) 96-well microplate and seal with non-permeable sealing film.
14. Store the plate at 2-8°C if storage is only for a few days. For long-term storage, samples should be kept at -20°C.

### Sample Location 
Post PCR Fridge in box "PAN_BDT_CNAT_psbA Box #2"